README.md

```diff
@@ -47,21 +43,17 @@ Various scripts and commands rely on environment variables being set. These are
 ### Make commands
 There are `make` commands that alias some of this functionality:
  * `lint` -- Lints the spec and code
- * `publish` -- Outputs the specification as a **single file** into the `build/` directory
- * `serve` -- Serves a preview of the specification in human-readable format
+ * `build-spec` -- Outputs the specification as a **single file** into the `build/` directory
+ * `run-spec-viewer` -- Serves a preview of the specification in human-readable format
  * `generate-examples` -- generate example objects from the specification
  * `validate` -- validate generated examples against FHIR R4
 
 ### Running tests
 #### End-to-end tests
 To run tests, you need to supply an environment. A `local` environment and an environment template are included under `tests/e2e/environments`.
 
-Set the following environment variables for local testing:
- * `ENVIRONMENT`: `local`
- * `API_TEST_ENV_FILE_PATH`: `tests/e2e/environments/local.postman_environment.json`
- * `API_TEST_URL`: `localhost:9000`
+In order for local tests to work, you must have the sandbox server running locally and have symlinked the mock files. Under sandbox/mocks there are some json files with the paths to symlink to as their content. For each of these symlink to these files, removing the original.
 
-In order for local tests to work, you must have the sandbox server running locally.
 ```
 make sandbox
```

Comment:
I think this should be make run-sandbox?

Reply:
Good spot, will update